sfyra/pkg/tests/server_class.go

```diff
@@ -39,8 +39,8 @@ import (
 )
 
 const (
-	defaultServerClassName  = "default"
-	workloadServerClassName = "workload"
+	defaultServerClassName = "default"
+	serverClassName        = "sfyra"
```

Comment:
using `sfyra` for all deployed clusters, as this allows to test with Talos `HEAD`